patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -129,3 +133,20 @@ function removeTrailingAndLeadingSlash (p: string) {
   }
   return p
 }
+async function filterAndCopyFiles (source: string, destination: string) {
+  const files = await packlist({ path: source })
+  if (files.length === 0) {
+    return source
+  }
```

Comment:
it would be nice to check if the list of file matches the files in the filesystem. If all the files should be included, then there is no reason to copy to a new directory.